modules/markup/reStructuredText/reStructuredText.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package rst
+
+import (
+	"bufio"
+	"bytes"
+
+	"code.gitea.io/gitea/modules/markup"
+
+	gorst "github.com/hhatto/gorst"
+)
+
+func init() {
+	markup.RegisterParser(Parser{})
+}
+
+// Parser implements markup.Parser for reStructuredText
+type Parser struct {
+}
+
+// Name implements markup.Parser
```

Comment:
I think function comment should tell what that function is doing, not which interface it fulfils.